### PR TITLE
Update upload to postgres files

### PR DESCRIPTION
**Pull request by avpeery**
### Why:
* This PR fixes bugs associated with the UploadToPostgres.py files in the ec2setup dir. This setup enables the ability to run these files to load data to local db. In addition, these changes also fix issues with foreign key naming conventions.

### CODE REVIEW

File: frontend/src/Components/AlgInputs/AlgInputsLoadForecast.js [hunk 1]
```diff
@@ -91,19 +93,53 @@ class AlgInputsLoadForecast extends Component {
         this.setState({ open: false});
     };
 
+    update = (field) => {
+        return e => this.setState({ [field]: e.currentTarget.value })
```
Comment (jongoncalves) [inline frontend/src/Components/AlgInputs/AlgInputsLoadForecast.js]: where is `e` coming from?

**Review by jongoncalves — APPROVED** [review]
There's a lot here, but it looks pretty good and like it's going in the right direction!

File: frontend/src/Components/AlgInputs/AlgInputsLoadForecast.js (hunk 1 above)
Comment (derins) [reply frontend/src/Components/AlgInputs/AlgInputsLoadForecast.js]: e is the synthetic event, specifically the on-change event of the current text input field we are updating.

Comment (jongoncalves) [reply frontend/src/Components/AlgInputs/AlgInputsLoadForecast.js]: Yes, I understand that, my point is, how is that being passed in? It seems like it's implicit, however, the binding of the arrow function looks odd to me and I'm surprised this works.
Here are some relevant resources:
https://reactjs.org/docs/handling-events.html#passing-arguments-to-event-handlers
https://stackoverflow.com/questions/42597602/react-onclick-pass-event-with-parameter

Either way, being explicit is better than implicit in this scenario.

Comment (avpeery) [reply frontend/src/Components/AlgInputs/AlgInputsLoadForecast.js:97]: ok so something like this, for explicit passing of e: 
`    update = (field, e) => {
        return e => this.setState({ [field]: e.currentTarget.value })
    };
`

Comment (derins) [reply frontend/src/Components/AlgInputs/AlgInputsLoadForecast.js]: @jongoncalves and @avpeery I pushed some changes

Comment (avpeery) [reply frontend/src/Components/AlgInputs/AlgInputsLoadForecast.js]: awesome! looks good on my end

**Review by derins — APPROVED** [review]

**Pull request merged**